extensions/src/poseHand/index.ts

```diff
@@ -0,0 +1,324 @@
+import { ArgumentType, BlockType, Extension, Block, DefineBlock, Environment, ExtensionMenuDisplayDetails, RuntimeEvent } from "$common";
+
+import Video from '../../../packages/scratch-vm/src/io/video';
```

Comment:
Hm, I'd rather not have to import something from the `scratch-vm` package. Looks like we only use: 

```
Video.FORMAT_IMAGE_DATA
```

^^^ What value does that correspond to? Can we stick it somewhere else?